propagation/onboarding.md

```diff
@@ -0,0 +1,75 @@
+# Algorithm Onboarding Service
+
+APEx offers an algorithm onboarding service as part of its propagation services to ensure that project workflows and algorithms remain
+accessible to the EO community beyond the project's completion. This service aims to sustain and promote the use of
+algorithms developed during ESA projects, ensuring their longevity and continued utilization.
+
+For engagement with stakeholders of the algorithm, APEx collaborates closely with
+the [Stakeholder Engagement Facility](https://esa-sef.eu/). This initiative will provide outreach and training towards
+users, to maximize the uptake of the onboarded algorithms.
+
+By leveraging APEx’s Algorithm Onboarding service, you can ensure that your project’s innovations continue to benefit
+the
+EO community long after your project ends.
+
+## Service Overview
+
+### Hosting Platform Onboarding Support
+
+APEx can help to identify the different hosting environments that the project can choose from based on criteria such as the chosen standard or the availability of specific datasets and processing capabilities that are needed by the project.
+APEx also assists in the onboarding process, ensuring that your algorithm is available for execution on the selected platform and ready for
+further integration into the APEx Algorithm Catalogue.
+
+:::{.callout-tip}
+A full list of the known APEx-compliant algorithm hosting platforms is available [here](index.qmd#what-are-the-platforms-to-choose-from).
+:::
+
+### Automated Testing and Benchmarking
+
+To maintain the integrity of the services, APEx conducts automated testing and benchmarking in collaboration with the
+respective project teams. This ensures that the algorithms remain functional and accessible over time. APEx offers this testing service for free, which reduces the maintenance burden on projects.
```

Comment:
To address  some valid concerns about how this testing can be perceived, I propose a more detailed explanation.
@Patrick1G  @p3dr0  your review would be appreciated.

```suggestion
The proper functioning of the service depends on a number of (external) factors, so APEx asks the project team to define at least one test scenario that verifies if the service is still functional. 

This has a number of benefits for involved parties:

* Algorithm maintainers get notified of issues, without having to build their own systems.
* Hosting platforms get notified of issues in a similar manner.
* APEx is able to maintain a catalog of working services, which ensures that the user base trusts the catalog as a source of working services, thus increasing the uptake of working services.
* When a platform offers a service on the NoR with an SLA, issues can be detected and thus resolved before a user notifies the NoR of an SLA breach. The hosting platform can also choose to use the automated test scenarios as a way to report on the functioning of the on-demand service.

APEx does not have a reporting procedure towards the NoR, or a requirement to report on the state of a service or a potential SLA breach, as APEx is also not involved in the SLA definition.

Service providers are not forced to take action in case of a negative test. APEx may simply use a visual clue to indicate a (temporary) issue, or may hide/remove services as part of regular catalog housekeeping when the service provider indicates that the problem will not be resolved or is simply no longer reachable.
```

Reply:
@jdries  - Added your suggestion to the text as it provides more useful information towards the reader.